### ISSUE TITLE
Different ports for artifacts and management

### ISSUE THREAD

**Issue by akasyanenko85** [eclipse/hawkbit#1431]
Hi
I Have already installed hawkbit-update-server
everything working

But question is
Is it possible to setup different ports for connection hawkbit-update clients and different port for management
?
Thank you

**Comment by avgustinmm** [comment]
No, its not possible. Both are on same Tomcat server that opens a single port.
If you need different ports you may put a reverse proxy in front of hawkBit and configure it properly.

**Comment by avgustinmm** [comment]
I'll close the issue since it's answered. 
Also, in future, please use stack overflow for asking questions as mentioned in hawkBit documentation - [Ask a Question](https://eclipse.dev/hawkbit/community/#ask-a-question)
